src/form-question-types.rst

```diff
@@ -2399,4 +2400,44 @@ storing the values for later use.
+
+The default audio settings minimize file size while maintaining reasonable quality for a conversation between two people. Audio files will be saved in the `amr` format with a bitrate of 12.2kbps and a sample rate of 8kHz, resulting in a file size of about 5MB per hour. This corresponds to the ``voice-only`` quality :ref:`for audio questions <customizing-audio-quality>`. You can override that default quality by specifying a value in the :th:`parameters` column in the same way as :ref:`for audio questions <customizing-audio-quality>`.
+
+While recording is ongoing, an audio status bar is shown at the top of the screen. This bar helps remind data collectors that they are being recorded and provides visual feedback about audio volume. If a data collector exits a form and then re-opens it for editing, audio recording is resumed. Audio recording continues as long as the form is open, even if another application is in the foreground or the screen is locked. Note that settings can't be accessed directly from a form that has background recording. The audio file sent to the server will include audio from every time that the form was opened for editing.
+
+.. tip::
+
+  Android devices can make many sounds during use and these will be included in recordings. We recommend turning off sounds from button presses, camera shutters and notifications before recording.
+
+It can be helpful to combine background audio recording with :doc:`audit logging <form-audit-log>` to have more context while listening to the recording. Recording starts at the ``form start`` and ``form resume`` events and stops at the ``form exit`` event. You can use the difference between the ``form start`` time and the start time of an event to identify how far into the background recording a certain event happened.
+
+There is an override available to data collectors when recording might compromise safety or when consent to record can't be obtained but it's still important to capture some data. When the audio recording option is unchecked, audio recording ends immediately and any previously-recorded audio is deleted. Recording can't be resumed during the current form-filling session. The setting persists across form filling sessions. If :doc:`audit logging <form-audit-log>` is enabled, a ``background audio disabled`` event will be logged if a data collector toggles off recording and a ``background audio enabled`` event will be logged if a data collector toggles it back on.
+
+In some rare cases such as the device running out of space, the recording may complete successfully but not be attached to the form. If this happens, the recording may be available in the ``recordings`` folder of the :ref:`Collect directory <collect-directory>`. This folder is never cleared so consider emptying it yourself once you have retrieved its files.
```

Comment:
Maybe this could be a warning? Or we could pull it out as a subsection of the recording question section and link to it here.